### DISCUSSION TITLE
Help: SK64 first working but not with big EasyFlash Crts, then Dead PLA, finally Black Screen with new pla.

### DISCUSSION THREAD

**Discussion by nosyn00b** [frntc/Sidekick64#98] [Q&A] [answered]
I recently built my SK64 from scratch and I had success :D. It all seemed to work as expected, at least at the beginning (CRTs, D64s, PRGs... menu system, screen animation.. all perfect)

I then discovered that EasyFlash CRTs bigger than 16 blocks (or perhaps all Easyflash?) did not work. G64 512K cartridges worked, and so also big ocean cartridges so it seemed not to be a memory-related problem.

After many tries the PLA on my dear Breadbin C64 suddenly died.... I replaced it with a robust AlphaPLA but the SK64 could now run only standard cartridges, nothing else worked. 

To be sure there was something wrong with the SK64, and not the new PLA, I got out my spare C64C. The result (with the same config) was simply a black screen, and then... everything died ... I suspect another PLA is dead (don't have a spare this time :-(  ).

Finally - also using my repaired and working old C64 - Raspberry boots, but the  C64 remains black screen when SK64 is inserted (the C64 is regularly working without SK64 and with other cartridges). On the HDMI only the Sidekick logo is displayed.

I need some help to dig out the problem. I suspect some faulty chip at this point, but which one?

Any idea where to look on the SK64 circuitry/contacts/chips considering these symptoms (first nonfunctioning BIG/EasyFLash CRTs, then fried PLAs (probably this happened when I continuously tried out big EasyFlash CRT) and finally black screen?

Thanks,

PS: my SK64 is the last PCB version and I mounted the last HDMI firmware, I'm using an external 5V 3A Power supply.

**Comment by frntc** [comment]
Sorry for your PLA(s)... there are many ways to damage them, but you're the first one (I know of) who did that with a SK64 :-/

If some bank switching cartridges work, and Easyflash doesn't it might be one of the IO1/2 lines, i.e. one of the LVC257 on the SK64 (check the center one first). The center 257 also connects directly to the PLA via ROML, but I think something must go very wrong to damage it this way.

A black screen can be caused by almost everything: any pulled address or data line can stop the CPU from working, also some wrongly set signal lines for configuring the PLA can prevent it wrong working.

The first thing I'd to is looking for solder bridges or other possible defects, if you don't spot anything replace the center LVC257, then check the other LVC245.  The LVC07 sets some control signals... for the rest we'd need quite a bit measuring at the expansion port...

**Comment by nosyn00b** [comment, marked as answer]
After all, my C64s broke for reasons that were not directly related to the SK64 usage.

You were perfectly right on the first try about the IO1/2 lines. I discovered that those lines were not at the right TTL levels because of defective LVC257s. This happened even on the second SK64 that I built (fearing my first SK64 was a PLA killer, that - at last - it wasn't)!

Thank you for all the support you offered.
